pages/area-chart/test.page.tsx

```diff
@@ -47,6 +48,19 @@ export default function () {
             highlightedSeries={highlightedSeries}
             onHighlightChange={onHighlightChange}
             xDomain={xDomain}
+            detailPopoverFooter={() => (
+              <Box margin={{ top: 'm' }}>
```

Comment:
I think unless we want different spacing for different types of content we expect it would be great to have some default. The "total" can be supported separately and for something more custom I'd opt for providing a function to render the entire popover content.

Reply:
Other than that, the code looks good 👍